src/Tizen.NUI/src/public/BaseComponents/ViewAccessibility.cs

```diff
@@ -283,18 +237,22 @@ public AddressCollection GetAccessibilityRelations()
             return result;
         }
 
+        ///////////////////////////////////////////////////////////////////
```

Comment:
could you resolve the warning above in line 218?
`second` should be null checked.